### DISCUSSION TITLE
Presence on User

### DISCUSSION THREAD

**Discussion by almostSouji** [discordjs/discord.js#5248] [Q&A] [answered]
Is there any good reason why we apply Presence back to the User instance? Presences are intrinsically guild bound data, yet we apply it back to the underlying User.

In day to day support, this has often caused issues to arise where the presence is not available on the User but works flawlessly if accessed via the GuildMember instance (`message.author` being but one example).

**Comment by MattIPv4** [comment, marked as answer]
Seems it is just a glorified helper getter to get the presence from the first guild that has the presence.

https://github.com/discordjs/discord.js/blob/f95f5dcd791b39c6a4d60dc8d64b0287e06ed768/src/structures/User.js#L141-L153

Seems like a sensible bit of cruft to remove given it causes confusion and is nothing more than a helper, no unique data stored.
